data-reconciliation-app/docs/README.md

```diff
@@ -0,0 +1,13 @@
+## Docs Folder 
+
+### Architecture Design Record (ADR)
```

Comment:
I always thought it was `Architecture Decision Record`

Reply:
It is! I must have been thinking about ADS while writing this doc. Thanks for catching this.